modelling/src/main/java/org/axonframework/modelling/saga/repository/AnnotatedSagaRepository.java

```diff
@@ -67,17 +67,18 @@
      * Instantiate a {@link AnnotatedSagaRepository} based on the fields contained in the {@link Builder}.
      * <p>
      * Will assert that the {@code sagaType}, {@link SagaStore} and {@link ResourceInjector} are not {@code null}, and
-     * will throw an {@link AxonConfigurationException} if any of them is {@code null}.
-     * Additionally, the provided  Builder's goal is to either build a {@link SagaModel} specifying generic {@code T} as
-     * the Saga type to be stored or derive it based on the given {@code sagaType}.
-     * All Sagas in this repository must be {@code instanceOf} this saga type.
+     * will throw an {@link AxonConfigurationException} if any of them is {@code null}. Additionally, the provided
+     * builder's goal is to either build a {@link SagaModel} specifying generic {@code T} as the Saga type to be stored
+     * or derive it based on the given {@code sagaType}. Same for the {@link MessageHandlerInterceptorMemberChain}. All
```

Comment:
Nit: I think the sentence works a little better like this:

```suggestion
     * or derive it based on the given {@code sagaType}. The same argument applies to the {@link MessageHandlerInterceptorMemberChain}. All
```

Reply:
This adjustment should be made on the `builder()` method JavaDoc too.

Reply:
done